src/config.ts

```diff
@@ -115,6 +115,7 @@ export interface SnowpackConfig {
     };
   };
   buildOptions: {
+    baseUrl: string;
```

Comment:
did this accidentally pull in from that other PR?

Reply:
Good catch! Fixed